internal/flagtypes/dryrun.go

```diff
@@ -0,0 +1,71 @@
+package flagtypes
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// ensure they conform to the interfaces
```

Comment:
nit: end sentences with `.`
```suggestion
// ensure they conform to the interfaces.
```